phpcs.xml.dist

```diff
@@ -7,10 +7,305 @@
 
   <rule ref="Drupal">
     <!-- TODO: those rules are disabled for now until we fix the coding standards for them. -->
```

Comment:
this section is now empty, so the TODO comment should be moved/copied to the new exceptions we make below.